modules/serverless-creating-subscriptions.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * serverless/knative_eventing/serverless-subscriptions.adoc
+
+[id="serverless-creating-subscriptions_{context}"]
+= Creating a Subscription
+
+You can create a Subscription to connect a Service or other event sink to a channel.
+The following procedure shows how to create a Subscription object that uses the default InMemoryChannel type channel.
+
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
+====
+
+.Prerequisites
+
+* You must have a current installation of xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#serverless-install-web-console_installing-openshift-serverless[{ServerlessProductName}], including Knative Serving and Eventing, in your {product-title} cluster. This can be installed by a cluster administrator.
+* If you do not have an existing sink that you wish to use, you can create a Service to use as a sink by following the documentation on xref:../../serverless/serving-creating-managing-apps.adoc#serving-creating-managing-apps[Creating and managing serverless applications].
+* You will need a channel to connect your Subscription to. For more information about creating an InMemoryChannel type channel, see xref:../../serverless/knative_eventing/serverless-channels.adoc#serverless-channels[Using channels with Knative Eventing].
+// * If you want to use other Channel types with Subscriptions...
+// TODO: Create docs for these, link Kafka etc?
+
+.Procedure
+
+. Create a Subscription object that uses the default InMemoryChannel type channel, by creating a YAML file containing the following:
++
+[source,yml]
+----
+apiVersion: messaging.knative.dev/v1
```

Comment:
at least with 1.8.0, subscription is still not "v1" yet, should be "v1beta1" for now.